index.js

```diff
@@ -18,44 +17,22 @@ const setupHook = (options) => {
   const cfn = new AWS.CloudFormation({ region: options.region });
   const lambda = new AWS.Lambda({ region: options.region });
 
-  const repo = () => {
-    const query = { access_token: options.token };
-
-    const config = {
-      json: true,
-      headers: {
-        'User-Agent': 'github.com/mapbox/stork',
-        'Content-Type': 'application/json'
-      }
-    };
-
-    const uri = `https://api.github.com/repos/${options.org}/${options.repo}`;
-
-    return got.get(`${uri}?${querystring.stringify(query)}`, config)
-      .then((data) => data.body.id);
-  };
-
-  return Promise.all([
-    repo(),
-    cfn.describeStacks({ StackName: `stork-${options.suffix}` }).promise()
-  ]).then((results) => {
-    const repoId = results[0];
-    const data = results[1];
-
-    const outputs = data.Stacks[0].Outputs;
-    const installationId = Number(outputs
-      .find((output) => output.OutputKey === 'GithubAppInstallationId')
-      .OutputValue);
-    const FunctionName = outputs
-      .find((output) => output.OutputKey === 'GatekeeperLambda')
-      .OutputValue;
-
-    return lambda.invoke({
-      FunctionName,
-      InvocationType: 'RequestResponse',
-      Payload: JSON.stringify({ repoId, installationId })
-    }).promise();
-  });
+  return cfn.describeStacks({ StackName: `stork-${options.suffix}` }).promise()
+    .then((data) => {
+      const outputs = data.Stacks[0].Outputs;
+      const installationId = Number(outputs
+        .find((output) => output.OutputKey === 'GithubAppInstallationId')
+        .OutputValue);
+      const FunctionName = outputs
+        .find((output) => output.OutputKey === 'GatekeeperLambda')
+        .OutputValue;
+
+      return lambda.invoke({
+        FunctionName,
+        InvocationType: 'RequestResponse',
+        Payload: JSON.stringify({ repo: options.repo, org: options.org, installationId: installationId })
```

Comment:
@rclark this `installationId` parameter is carried over from before. However, poking around the lambda function code I can't actually find where this is used -- that function seems to have the installation ID already set as `process.env.GITHUB_APP_INSTALLATION_ID`. Does that make this `installationId: installationId` totally extraneous, or am I missing something?

Reply:
This was probably me trying to plan for a single gatekeeper lambda that could add **any** repo to **any** github app installation. I think we want to end up with such a function, but maybe don't need it as part of stork.